nemo/collections/asr/modules/audio_modules.py

```diff
@@ -0,0 +1,676 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Optional, Tuple
+
+import torch
+
+from nemo.collections.asr.parts.preprocessing.features import make_seq_mask_like
+from nemo.collections.asr.parts.utils.audio_utils import db2mag
+from nemo.core.classes import NeuralModule, typecheck
+from nemo.core.neural_types import AudioSignal, FloatType, LengthsType, NeuralType, SpectrogramType
+from nemo.utils import logging
+from nemo.utils.decorators import experimental
+
+try:
+    import torchaudio
+
+    HAVE_TORCHAUDIO = True
+except ModuleNotFoundError:
+    HAVE_TORCHAUDIO = False
+
+
+__all__ = [
+    'AudioToSpectrogram',
+    'SpectrogramToAudio',
```

Comment:
can you move 'AudioToSpectrogram' and 'SpectrogramToAudio' to already existing "audio_preprocessing.py" and rename the current to something like "speech_enhancement_modules.py" or something in that spirit.